src/config/config.c

```diff
@@ -656,6 +656,37 @@ __wt_config_getones_none(
     return (0);
 }
 
+/*
+ * __wt_config_tokenizer --
+ *     String tokenizer into the given array.
+ */
+int
+__wt_config_tokenizer(
+  WT_SESSION_IMPL *session, const char *cval_str, const char *delimeter, uint64_t *parse_array)
+{
+    uint64_t value;
+    uint16_t counter;
+    char *str;
+    char *token;
+
+    str = NULL;
+    counter = 0;
+
+    WT_RET(__wt_strndup(session, cval_str, strlen(cval_str), &str));
+
+    token = strtok(str, delimeter);
```

Comment:
We have deliberately avoided strtok historically because it's considered to be a security risk:
https://stackoverflow.com/questions/5999418/why-is-strtok-considered-unsafe

Reply:
Removed strtok tokenizer and used list.